apps/riscv-tests/isa/rv64uv/viota.c

```diff
@@ -8,18 +8,18 @@
 #include "vector_macros.h"
 
 void TEST_CASE1() {
-  VSET(8, e8, m1);
-  VLOAD_U8(v1, 1, 0, 0, 1, 0, 0, 0, 1);
+  VSET(16, e8, m1);
+  VLOAD_8(v1, 1, 0, 0, 1, 0, 0, 0, 1);
```

Comment:
`viota.m` works on mask vectors, so the source bits should be packed. 
Something like this should work:

```c
void TEST_CASE1() {
  VSET(1, e8, m1);
  VLOAD_8(v1, 0b10001001);
  VSET(8, e8, m1);
  asm volatile("viota.m v2, v1");
  VCMP_U8(1, v2, 0, 1, 1, 1, 2, 2, 2, 2);
}

void TEST_CASE2() {
  VSET(1, e8, m1);
  VLOAD_8(v1, 0b10001001);
  VLOAD_8(v0, 0b11000111);
  VSET(8, e8, m1);
  VLOAD_8(v2, 0, 1, 2, 3, 4, 5, 6, 7);
  asm volatile("viota.m v2, v1, v0.t");
  VCMP_U8(2, v2, 0, 1, 1, 3, 4, 5, 1, 1);
}
```